android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -109,21 +117,16 @@ protected void triggerLayoutInflation() {
             mAssetDecimals =
                     getIntent().getIntExtra(Utils.ASSET_DECIMALS, Utils.ETH_DEFAULT_DECIMALS);
             mCoinType = getIntent().getIntExtra(Utils.COIN_TYPE, CoinType.ETH);
-
-            getBlockchainToken(() -> {});
-
             if (mAssetSymbol.equals("ETH")) {
                 mAssetLogo = "eth.png";
             }
+
+            mAssetNetwork = mWalletModel.getCryptoModel().getNetworkModel().getNetwork(mChainId);
```

Comment:
what happens if `mWalletModel == null`?

Reply:
I can add a null check,
but don't think we have any reports of `mWalletModel` being null after successful login and it shouldn't be null i believe. The culprit is [native services from core](https://github.com/brave/brave-browser/issues/27805). Either way, we should prioritise https://github.com/brave/brave-browser/issues/27887 because showing empty UI with return isn't gonna be a good UX either.

Reply:
done https://github.com/brave/brave-core/pull/17796/commits/d5b3c0cb77cd076d580376b289d469db9e2ee3c4